pkg/cmd/build/authprovider.go

```diff
@@ -23,20 +23,24 @@ import (
 	"github.com/docker/cli/cli/config/configfile"
 	"github.com/docker/cli/cli/config/types"
 	"github.com/moby/buildkit/session/auth"
+	"github.com/okteto/okteto/pkg/env"
 	oktetoLog "github.com/okteto/okteto/pkg/log"
 	"github.com/okteto/okteto/pkg/okteto"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 )
 
+const oktetoLocalRegistryStoreEnabledEnvVarKey = "OKTETO_LOCAL_REGISTRY_STORE_ENABLED"
```

Comment:
@jLopezbarb could we document this variable in this section? Also, that section needs some love, if you could redo it and document our main feature flag envvars as part of this work, it would be awesome:
https://www.okteto.com/docs/manifest/okteto-variables/#feature-flag-variables

Reply:
Created the [following ticket](https://github.com/okteto/docs/pull/672)